Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdaterContentStrategy.cs

```diff
@@ -119,7 +119,15 @@ public void Update(CancellationToken cancellationToken)
                 DebugLogger.Log("Content installed with errors, requesting repair");
 
                 var appRepairer = new AppRepairer(_context, _status);
-                appRepairer.CheckHashes = true;
+                if (_context.App.RemoteMetaData.GetContentSummary(latestVersionId, cancellationToken).Files.Length < 10000)
```

Comment:
Please replace `10000` with a constant. For instance `MAX_CHECK_HASHES_FILE_COUNT`

Reply:
Done